src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -1367,6 +1389,12 @@ namespace LowTempRadiantSystem {
                                 "System",
                                 "Sum",
                                 thisHydrSys.Name);
+            SetupOutputVariable("Zone Radiant HVAC Operation Mode",
+                                OutputProcessor::Unit::None,
+                                thisHydrSys.OperatingMode,
+                                "System",
+                                "Average",
+                                thisHydrSys.Name);
```

Comment:
Reporting an integer operating mode is a little fuzzy, but there is precedent, it's OK.

Reply:
Granted it's a bit weird, but it gives the user some idea what's going on.  That's why I changed cooling mode to -1.  So, if the output is less than 0, the user will know cooling took place.  It was also a help for verifying that the changeover delay was working and being interpreted inside the code properly.